_sources/governance.rst.txt

```diff
@@ -0,0 +1,169 @@
+
+With these definitions, an Institutional Partner is any recognized legal entity in any country that employs at least 1 Institutional Contributor or Institutional Council Member. Institutional Partners can be for-profit or non-profit entities.
+
+Institutions become eligible to become an Institutional Partner by employing individuals who actively contribute to The Project as part of their official duties. To state this another way, the only way for a Partner to influence the project is by actively contributing to the open development of the project, in equal terms to any other member of the community of Contributors and Council Members. Merely using Project Software in institutional context does not allow an entity to become an Institutional Partner. Financial gifts do not enable an entity to become an Institutional Partner. Once an institution becomes eligible for Institutional Partnership, the Steering Council must nominate and approve the Partnership.
+
+If at some point an existing Institutional Partner stops having any contributing employees, then a one year grace period commences. If at the end of this one year period they continue not to have any contributing employees, then their Institutional Partnership will lapse, and resuming it will require going through the normal process for new Partnerships.
+
+An Institutional Partner is free to pursue funding for their work on The Project through any legal means. This could involve a non-profit organization raising money from private foundations and donors or a for-profit company building proprietary products and services that leverage Project Software and Services. Funding acquired by Institutional Partners to work on The Project is called Institutional Funding. However, no funding obtained by an Institutional Partner can override the Steering Council. If a Partner has funding to do PySAL work and the Council decides to not pursue that work as a project, the Partner is free to pursue it on their own. However in this situation, that part of the Partner’s work will not be under the PySAL umbrella and cannot use the Project trademarks in a way that suggests a formal relationship.
+
+Institutional Partner benefits are:
+
+* Acknowledgement on the PySAL website and in talks.
+* Ability to acknowledge their own funding sources on the PySAL website and in talks.
+* Ability to influence the project through the participation of their Council Member.
+* Council Members invited to SciPy Developer Meetings.
+
+A list of current and former Institutional Partners is maintained at the landing page.
+
+Document history
+----------------
+https://github.com/pysal/pysal.github.io/tree/master/_sources/governance.rst.txt
```

Comment:
Should `governance.rst.txt` simply be `governance.rst` or `governance.md`?

Reply:
@slumnitz Does this still need attention?

Reply:
Since the history should be pointing to this discussion/ commits and the changes we made to the existing document, it should be pointing here? We could also leave this out completely and only start a Document history once we decide the text needs changes in future?